boards/pico/examples/pico_gpio_in_out.rs

```diff
@@ -66,7 +66,7 @@ fn main() -> ! {
     let mut led_pin = pins.led.into_push_pull_output();
 
     // Our button input
-    let button_pin = pins.bootsel.into_pull_down_input();
+    let button_pin = pins.gpio15.into_pull_down_input();
```

Comment:
Still doesn't fit the comment at the top of the file: "The button should cause the line to be grounded, as the input pin is pulled high internally by this example."